license-checks.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="Header">
+        <property name="fileExtensions" value="java"/>
+        <property name="ignoreLines" value="2"/>
```

Comment:
Did you consider [this](http://checkstyle.sourceforge.net/config_header.html#RegexpHeader) as an alternative?